src/Microsoft.Extensions.WebEncoders/Testing/HtmlTestEncoder.cs

```diff
@@ -18,6 +18,11 @@ public override int MaxOutputCharactersPerInputCharacter
 
         public override string Encode(string value)
         {
+            if (string.IsNullOrEmpty(value))
```

Comment:
Good point and I will certainly add the missing `null` check. Same for the other overloads in this class.

Turns out I need to check for `string.Empty` in the `Encode(TextWriter output, string value, int startIndex, int characterCount)` overload as well. [`TextEncoder.Encode(TextWriter output, string value)`](https://github.com/dotnet/corefx/blob/master/src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoder.cs#L161) calls that one.